data/test/test_integration/domain_with_null_active_loop.yml

```diff
@@ -0,0 +1,64 @@
+version: "3.1"
+
+intents:
+  - greet
+  - goodbye
+  - affirm
+  - deny
+  - mood_great
+  - mood_unhappy
+  - bot_challenge
+  - block
+
+entities:
+  - account_type
+
+slots:
+  account_type: 
+    type: categorical
+    influence_conversation: true
+    values:
+      - savings
+      - checking
+      - credit
+    mappings: 
+    - type: from_entity
+      entity: account_type
+    # uncomment below to make the story conflict happen
```

Comment:
This comment is confusing, why would a story conflict happen if you comment out conditions with the current bugfix?

Reply:
I forgot to remove that. I copied the domain content from the reproduce-bot.